sdk/core/Azure.Core/src/SerializableOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure
+{
+    /// <summary>
+    /// TODO
+    /// </summary>
+    public class SerializableOptions
+    {
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public bool IncludeReadOnlyProperties { get; set; } = true;
+
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public bool HandleAdditionalProperties { get; set; } = true;
+
+        /// <summary>
+        /// TODO
+        /// </summary>
+        public bool PrettyPrint { get; set; } = false;
```

Comment:
Did we consider mirroring SJT with [WriteIndented](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.writeindented?view=net-7.0#system-text-json-jsonserializeroptions-writeindented)?

Reply:
Its interesting in the description of what you wrote they say "true if JSON is pretty printed on serialization" indicating that `PrettyPrint` is a pretty well understood concept.

Reply:
But they chose not to use that term. `WriteIndented` seems more inline with what it's actually doing.

Reply:
+1 that this property would be more helpful and consistent with other .NET APIs if it described its intent.

Reply:
We can discuss the naming here for sure, but PrettyPrint = Indent + Newlines + spaces after `:`, etc

Reply:
What scenario motivates the need for this property?

Reply:
Its a very common scenario to generate a user friendly readable format for logs vs the smallest possible format for wire transfer perf reasons.